### DISCUSSION TITLE
DBeaver can't list databases other than the default postgres database

### DISCUSSION THREAD

**Discussion by RickAnderson-hub** [dbeaver/dbeaver#22006] [Q&A]
### Description

I used DBeaver to create a database, schema and a test table.  I persisted it and it succeeded.  I couldn't list the new database.  I refreshed the views and also went to the bottom right where one can list the databases and only the postgres database was there.  I tried again to create the database under the same name and got the SQL error that the database exists.  I opened a connection using IntelliJ Idea and the database was there.  I also listed the databases using psql and all was fine.  I concluded that DBeaver had to be at fault.  

### DBeaver Version

Community Edition 23.22

### Operating System

Linux 6.6.2-arch1-1

### Database and driver

PostgreSQL JDBC Driver 42.5.2

### Steps to reproduce

1. create and run a docker container with bitnami/postgresql:latest.  Use the configs in the JSON below.
2. Connect with DBeaver and right click at the databases and create a new DB
3. create a new schema
4. create a new table
5. Press save to persist

"PortBindings": {
			"5432/tcp": [
				{
					"HostIp": "0.0.0.0",
					"HostPort": "5432"
				}
			]
		}
"Mounts": [
		{
			"Type": "volume",
			"Name": "postgres_data",
			"Source": "/var/lib/docker/volumes/postgres_data/_data",
			"Destination": "/var/lib/postgresql/data",
			"Driver": "local",
			"Mode": "z",
			"RW": true,
			"Propagation": ""
		}

### Additional context

no tunnels or proxies uses.  
No extensions
works in IntelliJ and psql

**Comment by LonwoLonwo** [comment]
Hello @RickAnderson-hub 

Please check this setting in your connection settings:

![2023-11-27 10_59_28-DBeaver 23 1 1 - Employee](https://github.com/dbeaver/dbeaver/assets/45152336/2b1044d3-29a5-4623-ad04-c1392815f562)

**Comment by RickAnderson-hub** [comment]
Thank you.  That solves my problem
